definitions.yaml

```diff
@@ -0,0 +1,235 @@
+swagger: "2.0"
+info:
+  description: "This is an [Express](http://expressjs.com/) web application which implements some tasks to automate the provisioning of users in the Digital Citizenship Azure API management _developer portal_.\n\nThe aim is to automate some operations that would otherwise require the manual intervention of the APIs administrator: this lets developers start testing the API just after signing-up.\n\nUser's authentication takes place against an [Azure Active Directory B2C](https://azure.microsoft.com/en-us/services/active-directory-b2c/) tenant."
+  version: "0.1.0"
+  title: Digital citizenship portal
+
+paths: {}
+
+definitions:
+
+  #Returned by GET /subscriptions
+  #Returned by GET /subscriptions/:email
+  SubscriptionCollection: #TODO: add `nextLink` property?
```

Comment:
This is the definition for SubcriptionCollection in the [library](https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/apimanagement) from which the type comes:
```javascript
export interface SubscriptionCollection extends Array<SubscriptionContract> {
  nextLink?: string;
}
```
I am wondering: when the server returns a response containing such kind of type, what should the client receive? Or does such type makes sense only for the server, and the client would receive just an array as currently defined in the yaml file?

Reply:
if the server returns that same type unmodified, then that type should be defined here

Reply:
It seems that `nextLink ` contains information about pagination, but the method which returns the `SubscriptionCollection` object is invoked without any pagination option, so I suppose it's not used.